### ISSUE TITLE
Suggestion: external browser and cluster jewel stuff

### ISSUE THREAD

**Issue by RandomHero13** [Kyusung4698/PoE-Overlay#781]
hi,

would it be possible to deactivate the overwolf warning in the settings that "i'll open my external browser and might get taken out of the game"?
also i don't get taken out of the game which i really would like if i ctrl+click in the evaluate screen ;)
and then i'd prefer if i could choose that click will open the external browser and ctrl+click the internal.

regarding cluster jewels: less passives is better but the evalute function is looking for "or greater" (which is right for almost everything). would it be possible to add an exception for this and maybe other im/explicits "where less is better"?

mfg
thomas

ps.: some kind of flask macro would be nice, but that would be far in the grey area :P

**Comment by Kyusung4698** [comment]
Hi, thanks for creating an issue. The external browser warning is currently a mandatory feature of Overwolf which has been a direct result of most of their apps being run in fullscreen. I'm currently waiting on an additional flag to suppress this dialog completely.

On the cluster jewels topic. Do you know any other stats that are also relaying on this kind of filtering? Also you can actually achieve this way of filtering by reseting the value and than just ignoring the min value. But i assume you're talking about a way of doing this by default. 

Yeah, a Flask Makro wont be added but i may add later is an additional flask timer in the center of the screen so you can keep up your flasks without looking away of the actions on your screen.

**Comment by RandomHero13** [comment]
yes, i was talking about default settings. changing it when evaluating a jewel works, but is tedious.

i think there is another ticket where someone found that negative numbers (resistances in his case) are treated as absolute values
everthing with (immortal flesh belt -all res and -physical dmg taken per hit)
also everything with "reduced attributes" (brawn jewel, willowgift amulet for example).
